### PR TITLE
Add 7.17 branch to the doc build

### PR DESCRIPTION
**Pull request by bmorelli25**
* Creates a new `/stack/versions/7.17.asciidoc` file copied from `7.16.asciidoc`
* Updates `conf.yaml` to include the 7.17 branch

### CODE REVIEW

**Review by gtback — APPROVED** [review]

File: conf.yaml [hunk 1]
```diff
@@ -70,9 +70,9 @@ contents_title:     Elastic Stack and Product Documentation
 # The keys don't really matter, but by convention the are the same as the variable.
 variables:
   stackcurrent: &stackcurrent 7.16
-  stacklive: &stacklive [ master, 8.0, 7.16, 6.8 ]
+  stacklive: &stacklive [ master, 8.0, 7.17, 7.16, 6.8 ]
```
Comment (lcawl) [inline conf.yaml:73]: Should only one of 7.x be "live"?
```suggestion
  stacklive: &stacklive [ master, 8.0, 7.17, 6.8 ]
```

Comment (bmorelli25) [reply conf.yaml:73]: Because 7.16 is still current, I don't think so. I think we remove 7.16 when 7.17 releases.

Comment (lcawl) [reply conf.yaml:73]: That makes sense, thanks!

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: conf.yaml [hunk 2]
```diff
@@ -501,7 +501,7 @@ contents:
               - title:      JavaScript Client
                 prefix:     javascript-api
                 current:    7.16
-                branches:   [ {main: master}, 8.0, 7.16, 6.x, 5.x, 16.x ]
+                branches:   [ {main: master}, 8.0, 7.17, 7.16, 6.x, 5.x, 16.x ]
                 live:       [ main, 8.0, 7.16 ]
```
Comment (bmorelli25) [inline conf.yaml:505]:
```suggestion
                live:       [ main, 8.0, 7.17, 7.16 ]
```

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Pull request merged**